modules/is/templates/carbon-home/bin/wso2server.sh.erb

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 # ----------------------------------------------------------------------------
-#  Copyright 2005-2012 WSO2, Inc. http://www.wso2.org
+#  Copyright 2019 WSO2, Inc. http://www.wso2.org
```

Comment:
Don't change the Licence of templated files. They should be as same as the original from the product distribution.

Reply:
Fixed in commit ef9e3bdba4b04b0b8f0cdcb3c2adaeab4de663a7